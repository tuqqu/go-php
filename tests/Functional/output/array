[0 0 0 0 0]
[0 0 0 0 100]
100
5
[str1 str2 str3 str4]
[[0 1 2] [1 2 3]]
66 [0 1 2]
[0 1 2]
