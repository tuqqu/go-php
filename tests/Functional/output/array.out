[0 0 0 0 0]
[0 0 0 0 100]
100
5
[str1 str2 str3 str4]
[[0 1 2] [1 2 3]]
[0 1 2]
[66 1 2]
